code/modules/gear_presets/_select_equipment.dm

```diff
@@ -331,12 +331,7 @@
 /datum/equipment_preset/proc/spawn_rebel_shoes(mob/living/carbon/human/H)
 	if(!istype(H)) return
 	var/shoespath = pick(
-		/obj/item/clothing/shoes/black,
-		/obj/item/clothing/shoes/brown,
-		/obj/item/clothing/shoes/laceup,
-		/obj/item/clothing/shoes/leather,
-		/obj/item/clothing/shoes/combat,
-		/obj/item/clothing/shoes/swat,
+		/obj/item/clothing/shoes/marine/upp,
```

Comment:
Variety please!

Reply:
Hmm I'll see if I can wrangle Riot's workboot version.

Reply:
Or just give them a knife somewhere else, I think reducing the pool of interesting gear they could have over the requirement of a boot knife is kind of lame.